frontend/public/components/storage/create-pvc.tsx

```diff
@@ -162,23 +184,29 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
         </label>
         <div className="form-group">
           {this.state.accessModeRadios.map(radio => {
-            const disabled = !!storageClassAccessMode;
-            const checked = !disabled ? radio.value === accessMode : radio.value === storageClassAccessMode;
-            return (
-              <RadioInput
-                {...radio}
-                key={radio.value}
-                onChange={this.handleChange}
-                inline={true}
-                disabled={disabled}
-                checked={checked}
-                aria-describedby="access-mode-help"
-                name="accessMode"
-              />
-            );
+            let radioObj = null;
+            const disabled = storageClassAccessModes.indexOf(radio.value) !== -1 ? false : true;
+
+            storageClassAccessModes.forEach(mode => {
+              const checked = !disabled ? radio.value === accessMode : radio.value === mode;
+              radioObj = (
+                <RadioInput
+                  {...radio}
+                  key={radio.value}
+                  onChange={this.handleChange}
+                  inline={true}
+                  disabled={disabled}
+                  checked={checked}
+                  aria-describedby="access-mode-help"
+                  name="accessMode"
+                />
+              );
+            });
+
+            return radioObj;
           })}
           <p className="help-block" id="access-mode-help">
-            {storageClassAccessMode ? 'Access mode is set by storage class and cannot be changed.' : 'Permissions to the mounted drive.'}
+            Access mode is set by storage class and cannot be changed.
```

Comment:
Don't we need to handle the case where the PVC isn't using a storage class?

Reply:
@spadgett By default, the 1st option will be selected from the Storage Class's dropdown while creating a PVC and there is no option for not selecting any storage class. Attaching the screenshot for it. 
![pvc](https://user-images.githubusercontent.com/5517376/60325584-9b511c00-99c2-11e9-8de4-cdf2387ff040.png)

Or are you referring to some other page?

Reply:
AFAIK storage classes are not required in k8s. What happens if I don't have any storage classes set up? As an administrator, I can manually provision persistent volumes and use those.